### ISSUE TITLE
Unable to document knowledge 

### ISSUE THREAD

**Issue by Bo0g1eMaN** [jptrrs/HumanResources#101]
Whenever I send a pawn to document knowledge it goes through like normal until the job is finished, then an error is thrown and there is no book actually made, then the pawn goes off like normal. 

here's the hugslib log: https://gist.github.com/d4b63bfc0460e85521f964f35e3f729e

**Comment by jptrrs** [comment]
Thanks for reporting. 
Is Coco a prisoner? Are you getting this only with prisoners? If so, that would be a [known bug](https://github.com/jptrrs/HumanResources/wiki/Compatibility-Notes#known-issues) with Prison Labor.
But also, there's an error in your log during initialization, on line 3701, that is most probably related to some modded research project with an unexpected configuration. This could be completely unrelated to the document job issue, but it's worth investigating nonetheless. Could you try isolating it? It's [step 4](https://steamcommunity.com/workshop/filedetails/discussion/2119687603/2295094308095765898/), but you don't need to go beyond the initial menu screen. 
Once we know the offending mod I can track the issue.

**Comment by Bo0g1eMaN** [comment]
Coco is a colonist, I've tried to have my other colonist document knowledge they knew too but that resulted in the same thing. I'll try step 4, but ooh boy, that'll be a tough one with 200 mods loaded

**Comment by jptrrs** [comment]
You can try dividing it into smaller chunks. Load, say, 50 at a time and watch when the error pops up. When it does, take those last 50 and try 10 at a time. Eventually you'll zero the conflict in.

**Comment by Bo0g1eMaN** [comment]
This appears to be the culprit for whatever reason, never would have guessed this was causing the issue: https://steamcommunity.com/sharedfiles/filedetails/?id=2121189509

**Comment by jptrrs** [comment]
Ok, I checked it and found out this mod introduced recipes with invalid products. This caused our startup process to fail, and you ended up with that situation you described. Apparently, that mod's author did a poor job when adapting the original. I've notified him and also introduced something to prevent cases like this from happening again. The update is online now. I'm glad you stumbled on this, it made the mod a little thougher! Thanks!
